spec-0000/SPEC0_versions.py

```diff
@@ -124,31 +124,45 @@ def get_release_dates(package, support_time=plus24):
 
 # Print drop schedule
 
-rel = {}
-for name, releases in package_releases.items():
-    rel |= {
-        " ".join([name, str(ver)]): [dates["release_date"], dates["drop_date"]]
-        for ver, dates in releases.items()
-    }
-
+data = []
+for k, versions in package_releases.items():
+    for v, dates in versions.items():
+        data.append(
+            (
+                k,
+                v,
+                pd.to_datetime(dates["release_date"]),
+                pd.to_datetime(dates["drop_date"]),
+            )
+        )
 
-print("Saving drop schedule to schedule.md")
-with open("schedule.md", "w") as fh:
-    current_quarter = None
+df = pd.DataFrame(data, columns=["package", "version", "release", "drop"])
 
-    # Sort by drop date
-    rel = dict(sorted(rel.items(), key=lambda item: item[1][1]))
+df["quarter"] = df["drop"].dt.to_period("Q")
 
-    for package, dates in rel.items():
-        qt = pd.to_datetime(dates[1]).to_period("Q")
+dq = df.set_index(["quarter", "package"]).sort_index()
 
-        # If drop date is in a new quarter, write out a heading
-        if qt != current_quarter:
-            if current_quarter != None:
-                fh.write("\n")
-            fh.write(f'{str(qt).replace("Q", " – Quarter ")}:\n\n')
-            current_quarter = qt
 
-        fh.write(
-            f"- {dates[1].strftime('%d %b %Y')}: drop {package} (initially released on {dates[0].strftime('%b %d, %Y')})\n"
-        )
+print("Saving drop schedule to schedule.md")
+with open("schedule.md", "w") as fh:
+    for quarter in sorted(set(dq.index.get_level_values(0))):
+        fh.write("#### " + str(quarter).replace("Q", " - Quarter ") + ":\n\n")
+        fh.write("Recommend drop support for:\n\n")
```

Comment:
Maybe then just make it a heading? 
```suggestion
        fh.write("###### Recommend drop support for:\n\n")
```

Reply:
Let's try